src/components/ShoppingList/ShoppingList.js

```diff
@@ -29,13 +29,28 @@ const ShoppingList = () => {
     setItems(filteredItems);
   };
 
+  const updateItem = (edit, key) => {
```

Comment:
Maybe this is clearer?
```suggestion
  const updateItem = (updatedValue, key) => {
```